suite-common/message-system/config/config.v1.json

```diff
@@ -1,8 +1,70 @@
 {
     "version": 1,
-    "timestamp": "2024-11-29T00:00:00+00:00",
-    "sequence": 68,
+    "timestamp": "2024-12-19T00:00:00+00:00",
+    "sequence": 69,
     "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "!",
+                        "mobile": "24.12.1",
```

Comment:
We should allow newer version too. And maybe even older? But that sounds like a too much on an edge case.

Reply:
fixed, force pushed